### ISSUE TITLE
example plugin request json api

### ISSUE THREAD

**Issue by ghost** [ShootingKing-AM/ThreadedSockets#7]
is there any up to date example plugin, that does for example a json call?

**Comment by ShootingKing-AM** [comment]
No. You need to send and recv data and use stocks (IIRC there are json stocks) to interpret json :3

**Comment by ghost** [comment]
thx. ok i now tried your example in the documentation, to request the steam api. works. can i run multiple request at the same time for each player? i would save the "g_iThreadHandle" for each player to identify them later. correct? and it it non-blocking by default?

its not clear to me. do i create 1 socket, and run all requests on it, or i create 1 sockeet for each request? how can i identify "socket_send_t" for each player?

**Comment by ghost** [comment]
ok i found your post on alliedmods saying it is impossible. so one global handle, 1 socket, 1 request only possible. thats sad. so i must do a chain

**Comment by ghost** [comment]
i tried the latest binaries ThreadedSockets-v1.2_3 and got the same error as wopox, i used his friends build, which runs, but the server has crashed multiple times a day with seg fault. so sadly i will stop using it
btw my cpu went from usual 20 to 100 for hours. until my host killed it. nice
